prov/verbs/src/verbs_cq.c

```diff
@@ -330,10 +300,11 @@ static inline int fi_ibv_poll_outstanding_cq(struct fi_ibv_msg_ep *ep,
 		dlist_remove(&wre->entry);
 		util_buf_release(wre_pool, wre);
 		return ret;
-	} else {
-		ret = fi_ibv_handle_internal_signal_wc(cq, &wc);
-		return ret ? ret : 1;
 	}
+
+	/* Shouldn't go here */
+	assert(0);
+	return 0;
```

Comment:
Can we restructure this to avoid unreachable code and the extra indentation above, even if it increases the diff count for white space only?

Reply:
Ops, done. It stayed from my draft implementation